staging/src/k8s.io/apiserver/pkg/admission/plugin/policy/generic/policy_dispatcher.go

```diff
@@ -194,23 +188,67 @@ func (d *policyDispatcher[P, B, E]) Dispatch(ctx context.Context, a admission.At
 					Evaluator: hook.Evaluator,
 				})
 			}
+		}
+	}
 
-			// VersionedAttr result will be cached and reused later during parallel
-			// hook calls
-			_, err = versionedAttrAccessor.VersionedAttribute(matchGVK)
-			if err != nil {
-				return apierrors.NewInternalError(err)
-			}
+	if len(relevantHooks) > 0 {
+		extraPolicyErrors, statusError := d.delegate(ctx, a, o, versionedAttrAccessor, relevantHooks)
```

Comment:
have not checked: this change makes `d.delegate` not see the hooks that have `hook.Error` set. What is the impact of that change?

Reply:
@alexzielenski do you remember?

Reply:
I think @sttts pointed out in another comment that the policy dispatcher was not yet being used so there is no impact